pillar/apps/odlvideo.sls

```diff
@@ -50,6 +50,13 @@
 {% set rabbit_creds = salt.vault.cached_read("rabbitmq-{env}/creds/odlvideo".format(env=ENVIRONMENT), cache_prefix=minion_id) %}
 {% set ga_json = salt.vault.read('secret-odl-video/' ~ ENVIRONMENT ~ '/ga-keyfile-json').data.value|json %}
 
+schedule:
+  refresh_{{ app_name }}_credentials:
+    days: 14
+    function: state.sls
+    args:
+      - django.config
```

Comment:
For OVS, we'll need to move the .env file creation into a separate state file and then update the pillar to call that as part of the django.config state. As it stands right now it's part of the `django.install` step.